torch/onnx/_internal/fx/passes/fx_to_onnxscript.py

```diff
@@ -27,14 +27,12 @@
 
 @_beartype.beartype
 def _fx_node_to_onnx_message_formatter(
-    fn: Callable, args: Tuple[Any, ...], kwargs: Dict[str, Any]
+    fn: Callable,
+    diagnostic_context: diagnostics.DiagnosticContext,
+    node: torch.fx.Node,
+    *args,
+    **kwargs,
 ) -> str:
-    # TODO(bowbao): refactor signature to avoid checking length.
```

Comment:
I would update the comment to retain info on why *args is used here, since readers may not be aware of the length checking problem

Reply:
Not sure what to put here. I guess it's clearer to fill in docstring for the decorator later, so they know what is expected. wdyt?